app/jobs/bulk_update/placements/import_row_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module BulkUpdate
+  module Placements
+    class ImportRowJob < ApplicationJob
+      queue_as :bulk_update
+
+      def perform(placement_row)
```

Comment:
i think best practice is to pass an id rather than AR instance as a job param?

Reply:
You don't have to do this anymore thanks to `globalid`